pkg/systemd/service-details.jsx

```diff
@@ -207,7 +207,7 @@ class ServiceActions extends React.Component {
                                               this.setState({ dialogMaskedOpened: false });
                                           }} />
                 }
-                <DropdownKebab id="service-actions" title={ _("Additional actions") } className={this.props.disabled ? "disabled" : "" }>
+                <DropdownKebab id="service-actions" title={ _("Additional actions") } disabled={this.props.disabled}>
```

Comment:
this unfortunately does not work. It is patterfly3 component and it does not have any property for disabling it. Nor does PF4. But PF4 items have 'isDisabled' property, which does not work here...
So it needs to stay as was. Maybe adding there comment?

Reply:
Undid the changes and added the comment as a fixup.